web/app/components/workflow/nodes/_base/node.tsx

```diff
@@ -193,7 +193,7 @@ const BaseNode: FC<BaseNodeProps> = ({
           {
             data._iterationLength && data._iterationIndex && data._runningStatus === NodeRunningStatus.Running && (
               <div className='mr-1.5 text-xs font-medium text-primary-600'>
-                {data._iterationIndex}/{data._iterationLength}
+                {data._iterationIndex > data._iterationLength ? data._iterationLength : data._iterationIndex}/{data._iterationLength}
```

Comment:
We should check `data._iterationLength > 0` here, otherwise it could lead to a can't divided by 0 error.

Reply:
Here is the front-end's "/" symbol display, not a division operation.